parts/agentparams.t

```diff
@@ -85,14 +85,14 @@
       "type": "string"
     },
     "{{.Name}}osImageSKU": {
-      "defaultValue": "16.04-LTS",
+      "defaultValue": "18.04-LTS",
       "metadata": {
         "description": "OS image SKU."
       },
       "type": "string"
     },
     "{{.Name}}osImageVersion": {
-      "defaultValue": "16.04.201804050",
+      "defaultValue": "latest",
```

Comment:
Assume the thought is that latest with security patches is better than locking (which makes debugging easier?)

Reply:
this default value isn't actually being used, I just changed it to reflect reality (https://github.com/Azure/acs-engine/pull/4251/files#diff-f41c9478d3eabb30a6e28202cbfe7e18R103)